src/System.Xml.XmlDocument/src/System/Xml/Dom/XmlLoader.cs

```diff
@@ -523,12 +508,9 @@ private XmlAttribute LoadAttributeNodeDirect()
                 defattr.SetSpecified(false);
                 return defattr;
             }
-            else
-            {
-                attr = new XmlAttribute(r.Prefix, r.LocalName, r.NamespaceURI, _doc);
-                LoadAttributeValue(attr, true);
-                return attr;
-            }
+            attr = new XmlAttribute(r.Prefix, r.LocalName, r.NamespaceURI, _doc);
```

Comment:
I'm personally not a huge fan of removing these explicit else blocks, although it is functionally equivalent. I don't think we have any coding/style guidelines around this sort of thing.